skipchain/api.go

```diff
@@ -40,146 +40,132 @@ func NewClient() *Client {
 	return &Client{Client: onet.NewClient("Skipchain")}
 }
 
-// NewLocalClient takes a LocalTest in order
-func NewLocalClient(local *onet.LocalTest) *Client {
-	return &Client{Client: local.NewClient("Skipchain")}
-}
-
-// CreateRootControl creates two Skipchains: a root SkipChain with
-// maximumHeight of maxHRoot and an control SkipChain with
-// maximumHeight of maxHControl. It connects both chains for later
-// reference.
-func (c *Client) CreateRootControl(elRoot, elControl *onet.Roster, baseHeight, maxHRoot, maxHControl int, ver VerifierID) (root, control *SkipBlock, cerr onet.ClientError) {
-	log.Lvl2("Creating root roster", elRoot)
-	root, cerr = c.CreateRoster(elRoot, baseHeight, maxHRoot, ver, nil)
-	if cerr != nil {
-		return
+// StoreSkipBlock asks the cothority to store the new skipblock, and eventually
+// attach it to the 'latest' skipblock.
+//  - latest is the skipblock where the new skipblock is appended. If el and d
+//   are nil, a new skipchain will be created with 'latest' as genesis-block.
```

Comment:
`StoreSkipBlock` is the actual message being sent to the service. `CreateGenesis` is a convenience-method that handles some of the set up of the skipblock, and then calls `StoreSkipBlock`.